jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,144 @@
+#!groovy
+// by default build number is used as cluster to prevent clashes with possibly already existing clusters
+String clusterName = BUILD_NUMBER
+String awsRegion = params.AWS_REGION
+String baseDomain = "intlyqe.com"
+if (CLUSTER_NAME) {
+    clusterName = CLUSTER_NAME
+}
+String clusterInfoDirectory = "clusters/" + clusterName
+String installConfigDirectory = ""
+String rootWorkspaceDir = ""
+
+def err = null
+String adminPassword = ""
+
+node("cirhos_rhel7") {
+    timeout(120) { ansiColor("gnome-terminal") { timestamps {
+        try {
+            currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
+            currentBuild.description = "clusterName: ${clusterName}\nawsRegion: ${awsRegion}"
+            
+            stage("Configure aws cli") {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'iam-openshift4-credentials', accessKeyVariable: 'AWS_ACCESS_KEY_ID', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
```

Comment:
Should we not be passing the "credentialsId" in as a parameter? I know initially we will have it locked to a single account, but we will more than likely change this soon enough.